src/subcommand/wallet/inscribe.rs

```diff
@@ -133,6 +132,9 @@ impl Inscribe {
     match (self.file, self.batch) {
       (Some(file), None) => {
         parent_info = Inscribe::get_parent_info(self.parent, &index, &utxos, &client, chain)?;
+
+        postage = self.postage.unwrap_or(TransactionBuilder::TARGET_POSTAGE);
```

Comment:
Let's move the `unwrap_or(TransactionBuilder::TARGET_POSTAGE` outside of this match, so we do:

```
let postage = postage.unwrap_or(TransactionBuilder::TARGET_POSTAGE);
```